src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const statesArr = [];
```

Comment:
AFAIK, you're not a pirate 😄 
```suggestion
  const statesArray = [];
```